tests/jerry/regression-test-issue-1065.js

```diff
@@ -13,4 +13,10 @@
 // limitations under the License.
 
 new (new (new RegExp().constructor)().constructor)().constructor.prototype.toString()
-RegExp().constructor().constructor.prototype.compile(RegExp.prototype)
+
+try {
+  RegExp().constructor().constructor.prototype.compile(RegExp.prototype)
```

Comment:
minor style issue: missing semicolon at end of line

Reply:
Thanks! I've updated the PR.